core/domain/feedback_domain.py

```diff
@@ -19,11 +19,30 @@
 
 
 class FeedbackThread(object):
-    """Domain object for a feedback thread."""
+    """Domain object for a feedback thread.
+    
+    Attributes:
+        full_thread_id: str. The feedback thread ID.
+        exploration_id: str. The associated exploration's ID.
+        state_name: str. The name of the state associated with 
+            the feedback thread.
+        original_author_id: str. The ID of the original author.
+        status: string(select). The current status of the 
```

Comment:
`status: str. The current status of the thread. Should be one of core.storage.feedback.gae_models.STATUS_CHOICES.`